src/components/LoginPage/LoginPage.tsx

```diff
@@ -1,29 +1,144 @@
-import React from 'react'
-import { Grid , Paper } from '@material-ui/core'
-import { makeStyles } from '@material-ui/core/styles'
+import React from "react";
+import { Container, Grid, Paper } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import background from "../../images/5.png";
+// import styles from "./LoginPage.module.css";
+import Form from "../loginForm/form";
 
-
-const useStyles = makeStyles({
-    container : {
-        marginTop:"25vh",
-        backgroundColor:"lightblue"
-    }
-});
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: theme.palette.background.default,
+    padding: "1rem",
+    display: "flex",
+    justifyContent: "center",
+    justifyItems: "space-between",
+  },
+  container: {
+    backgroundColor: "rgb(255, 255, 255)",
+    width: "100%",
+    minHeight: "100vh",
+    overflow: "hidden",
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "center",
+    position: "absolute",
+  },
+  container__background: {
+    width: "100%",
+    height: "100%",
+    zIndex: 1,
+    position: "absolute",
+    background: "linear-gradient(-10deg, rgb(251, 255, 0), #f58621)",
+    clipPath:
+      "polygon( 50% 0%, 100% 0, 70% 40%, 69% 73%,26% 82%,0 100%,0% 70%,0% 35%,0 0)",
+  },
+  container__leftPanel: {
+    zIndex: 11,
+    color: "#fff",
+    fontFamily: 'Sansita Swashed", cursive',
+    "& h1": {
+      fontSize: "4.5vw",
+      fontFamily: "'Sansita Swashed', cursive",
+    },
+  },
+  container__rightPanel: {
+    height: "100vh",
+    display: "flex",
+    justifyContent: "center",
+    justifyItems: "center",
+  },
+  container__rightPanel__paper: {
+    width: "15rem",
+    height: "20rem",
+    zIndex: 55,
+    margin: "1rem",
```

Comment:
for margin and padding, you can use `theme.spacing` which returns multiple of 8 in px